_posts/2020-12-18-community-meeting-poll.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: posts
+author: Kyle Davis
+comments: true
+title: "2021 Open Distro for Elasticsearch Community Meetings"
+categories:
+- meetings
+feature_image: "https://d2908q01vomqb2.cloudfront.net/ca3512f4dfa95a03169c5a670a4c91a19b3077b4/2019/03/26/open_disto-elasticsearch-logo-800x400.jpg"
+---
+
+We've been having regular bi-weekly community meetings over Zoom at 10 am Pacific on Mondays for quite some time.  While there has certainly been good response and attendance, we recognize that this time could be less than ideal for those who live in timezones with a UTC greater than +4 (India and points west to the international dateline).
+
+Now through December 31, 2020, there is a poll over on [the Open Distro forums](https://discuss.opendistrocommunity.dev/) about times/days for these meetings. You can take part in this survey by [logging into the forum and voting on a day and time](https://discuss.opendistrocommunity.dev/t/2021-open-distro-community-meetings/4404). It only takes a second and it will really help us understand when we can reach the most community members.
```

Comment:
about times and days for these meetings.

It only takes a few seconds and will really help us understand how to reach the broadest range of community members.